ksy_schema.json

```diff
@@ -216,7 +224,29 @@
             }
           }
         },
-        "license": { "type": "string" },
+        "tags": {
+          "type": "array",
+          "description": "list of some identifiers that can give additional information about the format\n\nshould be written in `lowercase-kebab-case` and listed in alphabetical order\n\nshould be used only at the top level",
+          "items": { "type": "string" },
+          "examples": [
+            "android",
+            "archive",
+            "database",
+            "dos",
+            "executable",
+            "filesystem",
+            "firmware",
+            "linux",
+            "log",
+            "macos",
+            "media",
+            "windows"
```

Comment:
Too much of them. 2-3 will suffice.

Reply:
Actually, this is list of all existing tags, so that users do not introduce new tags, and preferably use existing ones